src/centos/scripts/start-marklogic.sh

```diff
@@ -173,6 +184,7 @@ elif [[ "${MARKLOGIC_INIT}" == "true" ]]; then
 
     log "Initialzing MarkLogic on ${HOSTNAME}."
 
+
```

Comment:
Is there a purpose for the new whitespaces on lines 174 and 187?

Reply:
no, that should be removed.

Reply:
removed